Assignment1/Shopping/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,11 @@
+eclipse.preferences.version=1
```

Comment:
Don't include this file in git commit. Add it into git ignore

Reply:
For the next PR.
- Follow coding and naming convention.
- Attach assignment in PR.